tests/subsys/disk/stm32_sdmmc/boards/stm32l496g_disco.overlay

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020 Amarula Solutions
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	sdmmc1 {
+		compatible = "st,stm32-sdmmc";
+		label = "sdmmc1";
+		clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000400>;
+		pwr-gpios = <&gpioe 12 GPIO_ACTIVE_HIGH>;
+		cd-gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;
```

Comment:
I'm not familiar with stm32l496g_disco, but a brief look at the schematics suggest that `PA15` is not the correct value for `cd-gpios`.

Reply:
I think I mentioned this in one of my earlier comments, but as far as I can tell the gpio it's connected to is on some peripheral that is not currently supported. I just picked the stm32l496g_disco because board is similar to what I have and it has sdcard. I don't actually have that board. I could make the card detect pin optional and have the driver assume the sdcard is present if the gpio is not set.